recipe/bld.bat

```diff
@@ -62,51 +59,58 @@ if "%PKG_NAME%" == "pytorch" (
   @REM For the main script we just build a wheel for so that the C++/CUDA
   @REM parts are built. Then they are reused in each python version.
   set "PIP_ACTION=wheel"
+  set "PIP_VERBOSITY=-vvv"
 )
 
-if not "%cuda_compiler_version%" == "None" (
-    set USE_CUDA=1
+set "BUILD_CUSTOM_PROTOBUF=OFF"
+set "USE_LITE_PROTO=ON"
 
-    @REM set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%desired_cuda%
-    @REM set CUDA_BIN_PATH=%CUDA_PATH%\bin
+@REM TODO(baszalmstra): Figure out if we need these flags
+SET "USE_ITT=0"
+SET "USE_NUMA=0"
 
-    set TORCH_CUDA_ARCH_LIST=5.0;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0+PTX
+@REM TODO(baszalmstra): There are linker errors because of mixing Intel OpenMP (iomp) and Microsoft OpenMP (vcomp)
+set "USE_OPENMP=OFF"
 
-    set TORCH_NVCC_FLAGS=-Xfatbin -compress-all
+@REM Use our Pybind11, Eigen, sleef
+set USE_SYSTEM_EIGEN_INSTALL=1
+set USE_SYSTEM_PYBIND11=1
+set USE_SYSTEM_SLEEF=1
 
+if not "%cuda_compiler_version%" == "None" (
+    set USE_CUDA=1
     set USE_STATIC_CUDNN=0
-    set MAGMA_HOME=%PREFIX%
-
     @REM NCCL is not available on windows
     set USE_NCCL=0
     set USE_STATIC_NCCL=0
 
-    set MAGMA_HOME=%LIBRARY_PREFIX%
+    @REM set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%desired_cuda%
+    @REM set CUDA_BIN_PATH=%CUDA_PATH%\bin
 
-    set "PATH=%CUDA_BIN_PATH%;%PATH%"
+    set "TORCH_CUDA_ARCH_LIST=5.0;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0+PTX"
+    set "TORCH_NVCC_FLAGS=-Xfatbin -compress-all"
 
+    set MAGMA_HOME=%LIBRARY_PREFIX%
+    set "PATH=%CUDA_BIN_PATH%;%PATH%"
     set CUDNN_INCLUDE_DIR=%LIBRARY_PREFIX%\include
-
+    @REM turn off very noisy nvcc warnings
+    set "CUDAFLAGS=-w --ptxas-options=-w"
```

Comment:
```suggestion
    set "CMAKE_CUDA_FLAGS=-w --ptxas-options=-w"
```

I think it works. The new flags are passed in "libtorch" build, and according to `diff` that's the only change in CUDA invocations. I'll know for sure when libtorch recompiles and it starts building pytorch.